src/ape/api/accounts.py

```diff
@@ -137,6 +143,15 @@ def transfer(
         if value:
             txn.value = self._convert(value, int)
 
+        if not value:
+            assert (
```

Comment:
No need for these assert statements. It will fail naturally if this is not true.

Asserts in python should really only be used in tests.

Reply:
Also, these get stripped out with the -I flag so its not reliable

Reply:
should these be if statements then?

Reply:
Use ValueError

Reply:
if I don't have the assert statements then it doesn't process the transaction properly in the tests: 
`FAILED tests/functional/test_accounts.py::test_transfer_without_value - KeyError: 'send_everything'`
so i believe it needs to throw to provide those errors?

Reply:
ok i got it to work with valueError :D